modules/k8s_resource/k8s-resource.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "https://app.runx.dev/modules/k8s-resource",
+  "description": "Kubernetes Resource",
```

Comment:
I wonder if we should name this `k8s-manifest`
- a manifest file can define multiple resources, so `k8s-resource` (singular) could be misleading in that way
- consistent with terraform naming: better for terraform users
what do you think?

Reply:
sure, will rename

Reply:
done